src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color.js";
+import FONTFAMILY from "../../../variables/font_family.js";
+import TEXT from "../../../variables/texts";
+
+export const Input = ({ onEditComplete, defaultValue }) => {
+  const inputRef = useRef(null);
+
+  const onkeypress = (e) => {
+    if (e.key === "Enter") {
+      inputRef.current.onblur();
+      inputRef.current.onkeypress("blur", () => {
+        onEditComplete(inputRef.current.value);
+        onkeypress(inputRef.current.value);
+      });
+    }
+  };
+
+  useEffect((props) => {
```

Comment:
```suggestion
  useEffect(() => {
```
useEffect に渡す関数は引数を取りません。
ドキュメント：https://ja.react.dev/reference/react/useEffect